diesel/src/pg/upsert/on_conflict_extension.rs

```diff
@@ -0,0 +1,85 @@
+pub use super::on_conflict_clause::*;
+
+/// Adds extension methods related to PG upsert
+pub trait OnConflictExtension {
+    /// Adds `ON CONFLICT DO NOTHING` to the insert statement, without
+    /// specifying any columns or constraints to restrict the conflict to.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # #[macro_use] extern crate diesel;
+    /// # #[macro_use] extern crate diesel_codegen;
+    /// # include!("src/doctest_setup.rs");
+    /// #
+    /// # table! {
+    /// #     users {
+    /// #         id -> Integer,
+    /// #         name -> VarChar,
+    /// #     }
+    /// # }
+    /// #
+    /// # #[derive(Clone, Copy, Insertable)]
+    /// # #[table_name="users"]
+    /// # struct User<'a> {
+    /// #     id: i32,
+    /// #     name: &'a str,
+    /// # }
+    /// #
+    /// # fn main() {
+    /// #     use self::users::dsl::*;
+    ///     use self::diesel::pg::upsert::*;
```

Comment:
This line is displayed in the rendered documentation, and thus is the right level of indentation (There is a removed `# `)